earth_enterprise/rpms/build.gradle

```diff
@@ -262,13 +263,14 @@ task openGeeCommonRpm(type: GeeRpm, dependsOn: openGeeSharedFiles) {
     autoFindProvides = true
     autoFindRequires = true
 
-    requires('opengee-postgis', '1.5.8', GREATER | EQUAL)
     requires('chkconfig')
     requires('shadow-utils')
     requires('perl')
     requires('sudo')
-    conflicts('opengee-postgis', '2.0', GREATER | EQUAL)
-
+    requires('xerces-c')
+    requires('ogdi')
```

Comment:
I don't think this is necessary. It is auto-detected:

```bash
rpm -qpR opengee-common-5.2.0-1.el7.x86_64.rpm | grep ogdi

libogdi.so.3()(64bit)
```

Reply:
(see below)